docs/samples/Microsoft.ML.Samples/Dynamic/IidSpikeDetectorTransform.cs

```diff
@@ -83,5 +87,84 @@ public static void IidSpikeDetectorTransform()
+        {
+            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
+            // as well as the source of randomness.
+            var ml = new MLContext();
+
+            // Generate sample series data with a spike
+            const int size = 10;
+            var data = new List<IidSpikeData>(size);
+            for (int i = 0; i < size / 2; i++)
+                data.Add(new IidSpikeData(5));
+            // This is a spike
+            data.Add(new IidSpikeData(10));
+            for (int i = 0; i < size / 2; i++)
+                data.Add(new IidSpikeData(5));
+
+            // Convert data to IDataView.
+            var dataView = ml.CreateStreamingDataView(data);
+
+            // Setup IidSpikeDetector arguments
+            string outputColumnName = "Prediction";
+            string inputColumnName = "Value";
+            var args = new IidSpikeDetector.Arguments()
+            {
+                Source = inputColumnName,
+                Name = outputColumnName,
+                Confidence = 95,                // The confidence for spike detection in the range [0, 100]
+                PvalueHistoryLength = size / 4  // The size of the sliding window for computing the p-value
+            };
+
+            // The transformed model.
+            ITransformer model = new IidSpikeEstimator(ml, args).Fit(dataView);
+
+            // Create a time series prediction engine from the model.
+            var engine = model.CreateTimeSeriesPredictionFunction<IidSpikeData, IidSpikePrediction>(ml);
+            for (int index = 0; index < 5; index++)
+            {
+                // Anomaly spike detection.
+                var prediction = engine.Predict(new IidSpikeData(5));
+                Console.WriteLine("{0}\t{1}\t{2:0.00}\t{3:0.00}", 5, prediction.Prediction[0],
+                    prediction.Prediction[1], prediction.Prediction[2]);
+            }
+
+            // Spike.
+            var spikePrediction = engine.Predict(new IidSpikeData(10));
+            Console.WriteLine("{0}\t{1}\t{2:0.00}\t{3:0.00}", 10, spikePrediction.Prediction[0],
+                spikePrediction.Prediction[1], spikePrediction.Prediction[2]);
+
+            // Checkpoint the model.
+            var modelPath = "temp.zip";
+            engine.CheckPoint(ml, modelPath);
+
+            // Load the model.
+            using (var file = File.OpenRead(modelPath))
+                model = TransformerChain.LoadFrom(ml, file);
+
+            for (int index = 0; index < 5; index++)
+            {
+                // Anomaly spike detection.
+                var prediction = engine.Predict(new IidSpikeData(5));
+                Console.WriteLine("{0}\t{1}\t{2:0.00}\t{3:0.00}", 5, prediction.Prediction[0],
+                    prediction.Prediction[1], prediction.Prediction[2]);
+            }
+
+            // Prediction column obtained post-transformation.
+            // Datat Alert   Score   P-Value
```

Comment:
>Datat Alert   Score   P-Value [](start = 15, length = 29)

Did we spell anywhere in our documentation what this Alert, Score, and P-Value means?

Imagine person without statistical background who will try to run this Spike prediction.
All he wants is predict DDOS attack based on current traffic numbers.
Is he need to look on Alert column only?
What Score column for?
Should he look into P-Value column? (What is our null hypothesis here? Absence of spike or presence of spike?)